frontend/src/components/dropdown-menu/item-dropdown-menu.js

```diff
@@ -224,7 +223,7 @@ class ItemDropdownMenu extends React.Component {
                       <i className="sf3-font-down sf3-font rotate-270"></i>
                     </DropdownToggle>
                     <DropdownMenu
-                      positionFixed={true}
+                      className="position-fixed"
```

Comment:
reactstrap 9.x Dropdown not support positionFixed props, so use className instead